app/models/legal_aid_application.rb

```diff
@@ -484,6 +484,10 @@ def transactions_total_by_type(transaction_type, category_id)
     __send__("#{transaction_type}_transactions").amounts.fetch(category_id, 0)
   end
 
+  def regular_transaction_category(transaction_type)
+    regular_transactions.where(transaction_type:)
+  end
```

Comment:
is this method being used? can't see it called anywhere. If not I would remove. If it is, then a minor nit is that  it is returning a collection of results too, rather than a single category, as implied by the singular name - regular_transaction_category -> regular_transaction_categories